src/TestSuite/TestCase.php

```diff
@@ -50,20 +51,23 @@ abstract class TestCase extends BaseTestCase
 {
     use LocatorAwareTrait;
 
-    /**
-     * The class responsible for managing the creation, loading and removing of fixtures
-     *
-     * @var \Cake\TestSuite\Fixture\FixtureLoader|null
-     */
-    public $fixtureManager;
-
     /**
      * Fixtures used by this test case.
      *
      * @var array<string>
      */
     protected $fixtures = [];
 
+    /**
+     * @var class-string<\Cake\TestSuite\Fixture\FixtureStrategyInterface>
+     */
+    protected string $fixtureStrategyClass = TruncateFixtureStrategy::class;
+
+    /**
+     * @var class-string<\Cake\TestSuite\Fixture\SchemaStrategyInterface>
+     */
+    protected string $schemaStrategyClass = CakeSchemaStrategy::class;
```

Comment:
I don't think this is a good default for applications. It is going to make a lot of work for application developers who never will want to use a big array to contain their schema definitions.